core/src/test/java/io/confluent/rest/Http2Test.java

```diff
@@ -78,6 +80,19 @@ public class Http2Test {
   private static final String SSL_PASSWORD = "test1234";
   private static final String EXPECTED_200_MSG = "Response status must be 200.";
 
+  HttpClient httpClient(SslContextFactory.Client sslContextFactory, HTTP2Client http2Client) {
```

Comment:
is there a reason to duplicate httpClient() in [ErrorHandlerIntegrationTest.java](https://github.com/confluentinc/rest-utils/pull/552/files/3f66161d8627842ec24f92c5ca181b3f1c43e602..ae462b74e51e58aecd263b8432f11bb1da40b819#diff-9de9ec813d1515d128281c576ab6dde4e5fc3f92e14757325294b2123cb558e4)? you an put the method as static func in utils class ex - core/src/test/java/io/confluent/rest/TestUtils.java?